dobson/utils.py

```diff
@@ -0,0 +1,62 @@
+import configparser
+import json
+import os
+import subprocess
+from collections import namedtuple
+
+
+Device = namedtuple('Device', ['presence', 'user', 'model'])
+
+config = configparser.ConfigParser()
+config.read('dobson/config.ini')
+
+
+SLACK_API_TOKEN = config['dobson']['SlackApiToken']
+
+# Only allow querying for presence in certain channels
+ALLOWED_CHANNEL_IDS = json.loads(config['dobson']['AllowedChannelIds'])
+
+# The user ID of the dobson bot
+DOBSON_SLACK_ID = config['dobson']['DobsonSlackId']
+
+# file to log current MAC Addresses in
+MAC_LOG_FILE = config['dobson']['MacAddressLogFile']
+
+# create KNOWN_DEVICES, which maps mac addresses (str) to Device instances
+if not os.path.isfile(config['dobson']['KnownDevicesFile']):
```

Comment:
If there's both paths in an `if` statement, I think putting the positive one first makes it clearer (the one without `not`), so that it creates the file in the `else` statement. Not a big deal or anything, I just find it easier to read, but I could see arguments on this either way.